src/mixins/portfolio.js

```diff
@@ -547,27 +558,13 @@ export const createPortfolioMixin = ({
       }
     },
     async loadNFTListByAddress(address) {
-      const blockingPromises = [
-        this.fetchNFTDisplayStateListByAddress(address),
-      ];
-      if (this.isCurrentTabCollected) {
-        blockingPromises.push(this.fetchCollectedNFTClassesByAddress(address));
-        this.fetchCreatedNFTClassesByAddress(address);
-      } else if (this.isCurrentTabCreated) {
-        blockingPromises.push(this.fetchCreatedNFTClassesByAddress(address));
-        this.fetchCollectedNFTClassesByAddress(address);
-      }
-
-      if (
-        (this.isCurrentTabCollected &&
-          !this.getCollectedNFTClassesByAddress(address)) ||
-        (this.isCurrentTabCreated &&
-          !this.getCreatedNFTClassesByAddress(address))
-      ) {
-        this.isLoading = true;
-        await Promise.all(blockingPromises);
-      }
-      this.isLoading = false;
+      this.fetchCollectedNFTClassesByAddress(address).then(() => {
+        this.isLoadingCollectedNFTClasses = false;
+      });
+      this.fetchCreatedNFTClassesByAddress(address).then(() => {
+        this.isLoadingCreatedNFTClasses = false;
+      });
```

Comment:
Just a question: How complex would this become if we only fetch collect/created when the related tab is shown?

Reply:
Can do but haven't looked into how we can trigger fetching from UI. Here we concurrently load both but do not block each other, because I think it's common that users may switch between collected/created tabs. Collected can be preloaded when a user is checking created.

Reply:
Hitting collected api when viewing created tab affects performance of created tab hugely though? In cases like ckxpress and edmondyu

Reply:
Do you mean performance is affected by fetching the collected list? 
Currently, the UI will load aggregated info and display when the created list is ready, and won't regard whether the collected list is still fetching or is ready.